### PR TITLE
Add lobby chat button.

### PR DESCRIPTION
**Pull request by corydalis10**

### CODE REVIEW

File: src/styles/globals.css [hunk 1]
```diff
@@ -10,6 +10,20 @@ html {
   background-color: #101010;
 }
 
+.if-mobile {
```
Comment (goffxnca) [inline src/styles/globals.css]: bro for responsive please dont use low-level css media screen, tailwind already provided us that functionality: https://tailwindcss.com/docs/responsive-design

**Review by goffxnca — CHANGES_REQUESTED** [review]

File: src/styles/globals.css (hunk 1 above)
Comment (ghost) [reply src/styles/globals.css]: But if i use tailwind, i can't do that.

Comment (goffxnca) [reply src/styles/globals.css]: tailwind responstive is mobile first bro. so anything that is not mobile you can prefix sth like  hidden md:block effectively this whole thing will mean on mobile hide it and md(ipad) and above show it.

**Review by goffxnca — APPROVED** [review]
all good now bro.

**Pull request merged**